modules/mail/assign.js

```diff
@@ -145,6 +146,39 @@ if (constants.canSendEmail) {
     )
   }
 
+  AssignMail.notifyInterestedUser = (user, task, assignedUser) => {
+    const to = user.email
+    const language = user.language || 'en'
+    const name = user.name || user.username
+    const assignedUserName = assignedUser.name || assignedUser.username
+    i18n.setLocale(language)
+    setMomentLocale(language)
+    request(
+      to,
+      i18n.__('mail.interested.user.assigned.subject'),
+      [
+        {
+          type: 'text/html',
+          value: `
+           <p>${i18n.__('mail.hello', { name: name })}</p>
+           <p>${i18n.__('mail.interested.user.assigned.main', { username: assignedUserName, title: task.title, url: `${process.env.FRONTEND_HOST}/#/task/${task.id}` })}</p>
+          ${Signatures.buttons(language, {
+    primary: {
+      label: 'mail.assigned.end.owner.button.primary',
+      url: `${process.env.FRONTEND_HOST}/#/task/${task.id}`
+    },
+    secondary: {
+      label: 'mail.assigned.end.owner.button.secondary',
+      url: `${task.url}`
+    }
+  })}
```

Comment:
```suggestion
          
```

We can remove here, cause these buttons was designed for notifications to the author, so we will not need here.